src/test/java/org/isf/utils/file/TestFileTools.java

```diff
@@ -531,4 +536,39 @@ public void testHumanReadableByteCount() throws Exception {
 		assertThat(FileTools.humanReadableByteCount(268435456L, Locale.US)).isEqualTo("256.0 M");
 		assertThat(FileTools.humanReadableByteCount(1073741824L, Locale.US)).isEqualTo("1.0 G");
 	}
+
+	@Test
+	public void testReadFileToStringLineByLineWithHtml() {
+		File file = getFile("testFile.txt");
+		String result = FileTools.readFileToStringLineByLine(file.getAbsolutePath(), true);
+
+		assertNotNull(result);
+		assertTrue(result.startsWith("<html>"));
+		assertTrue(result.endsWith("</html>"));
+		assertTrue(result.contains("<br>"));
+		assertTrue(result.contains("This could be one line text"));
+		assertTrue(result.contains("or text with"));
```

Comment:
Shouldn't this block really be:


assertThat(result).isNotNull();
assertThat(result).startsWith("\<html\>");
assertThat(result).endsWith("\</html\>");
assertThat(result).contains("\<br\>");
assertThat(result).contains("This could be one line text");
assertThat(result).contains("or text with");

Same thing for the next test too.

Reply:
you're right